app/screens/DashboardEditScreen.tsx

```diff
@@ -1,28 +1,90 @@
 import { FC } from "react"
+import { useState } from "react"
 import { Pressable } from "react-native"
 import { View } from "react-native"
+import { TextStyle } from "react-native"
+import DraggableFlatList, { RenderItemParams } from "react-native-draggable-flatlist"
 
+import { DashboardSettingsCard } from "@/components/DashboardSettingsCard"
 import { Icon } from "@/components/Icon"
 import { Screen } from "@/components/Screen"
 import { Text } from "@/components/Text"
 import { AppStackScreenProps } from "@/navigators/AppNavigator"
 import { useAppTheme } from "@/theme/context"
+import { spacing } from "@/theme/spacing"
 import { $topRightIcons, $headerIcons } from "@/theme/styles"
+import type { ThemedStyle } from "@/theme/types"
+
+const initialCards = [
+  { id: "duty", title: "Who's on duty", subtitle: "See who's on duty based on your saved filters" },
+  {
+    id: "upcomingShift",
+    title: "Upcoming shifts",
+    subtitle: "View your scheduled shifts for this week",
+  },
+  {
+    id: "upcomingLeave",
+    title: "Upcoming leaves",
+    subtitle: "Track your approved and pending leaves for this month",
+  },
+  {
+    id: "openShift",
+    title: "Open shifts",
+    subtitle: "Browse available shifts you can apply for this week",
+  },
+  { id: "roster", title: "Team Roster", subtitle: "See Team Roster with your saved preferences" },
+]
 
 export const DashboardEditScreen: FC<AppStackScreenProps<"EditDashboard">> =
   function DashboardEditScreen(_props) {
     const { navigation } = _props
     const { themed } = useAppTheme()
 
+    const [cards, setCards] = useState(initialCards)
+    const [checkedStates, setCheckedStates] = useState<Record<string, boolean>>({})
+
+    const toggleChecked = (id: string, newChecked: boolean) => {
+      setCheckedStates((prev) => ({ ...prev, [id]: newChecked }))
+    }
+
+    const renderItem = ({ item, drag }: RenderItemParams<(typeof cards)[0]>) => {
+      return (
+        <DashboardSettingsCard
+          key={item.id}
+          title={item.title}
+          subtitle={item.subtitle}
+          checked={!!checkedStates[item.id]}
+          onToggle={(newChecked) => toggleChecked(item.id, newChecked)}
+          onDrag={drag}
+        />
+      )
+    }
+
     return (
-      <Screen preset="scroll" safeAreaEdges={["top"]}>
+      <Screen preset="fixed" safeAreaEdges={["top"]}>
         <View style={themed($topRightIcons)}>
           <Pressable onPress={() => navigation.goBack()} style={themed($headerIcons)}>
             <Icon icon="anchor" />
           </Pressable>
         </View>
-        <Text preset="heading">Dashboard Editor Screen</Text>
+
+        <Text style={themed($editDashboardText)}>Customise what you see on your dashboard</Text>
+
+        <DraggableFlatList
+          data={cards}
+          keyExtractor={(item) => item.id}
+          renderItem={renderItem}
+          onDragEnd={({ data }) => setCards(data)} // update card order
+        />
```

Comment:
OKAY I did some testing on your branch, dont change anything and just add the following 2 props I think

```suggestion
        <DraggableFlatList
          data={cards}
          keyExtractor={(item) => item.id}
          renderItem={renderItem}
          onDragEnd={({ data }) => setCards(data)}
          scrollEnabled={false}    // This so the list itself isnt draggable (so screen and list should be fixed)
          activationDistance={Number.MAX_SAFE_INTEGER}    // Stop the dragging functionality (bit scuffed)
        />
```